plutus-tx/src/Language/PlutusTx/Applicative.hs

```diff
@@ -29,7 +32,15 @@ a1 *> a2 = (id <$ a1) <*> a2
 (<*) :: Applicative f => f a -> f b -> f a
 (<*) = liftA2 const
 
+{-# INLINABLE sequence #-}
+-- | Sequence a list of applicative actions.
+sequence :: Applicative f => [f a] -> f [a]
+sequence []    = pure []
+sequence (h:t) = (:) <$> h <*> sequence t
```

Comment:
I'd provide `traverse` and define `sequence` as `traverse id`.

Reply:
Yes, sure.